test-tests/shared/src/test/scala/zio/test/environment/ConsoleSpec.scala

```diff
@@ -0,0 +1,82 @@
+package zio.test.environment
+
+import zio.test.environment.TestConsole.Data
+import zio.test.Assertion._
+import zio.test._
+
+object ConsoleSpec
```

Comment:
that simplifies things a lot! I've left "reads from input" as is otherwise it would be identical to "feeds lines to input" and would fail if `feedLines` broke